src/components/general/dateRangePicker.tsx

```diff
@@ -23,14 +25,17 @@ const DateRangePicker = ({
       onChange={
           (item) => {
             if (item.selection.startDate && item.selection.endDate) {
-              setCurrDates([item.selection.startDate, item.selection.endDate]);
+              setDates([
+                item.selection.startDate.toString(),
+                item.selection.endDate.toString(),
+              ]);
             }
           }
       }
       moveRangeOnFirstSelection={false}
       ranges={[{
-        startDate: currDates.length > 0 ? currDates[0] : new Date(),
-        endDate: currDates.length > 0 ? currDates[1] : new Date(),
+        startDate: dates.length > 0 ? new Date(dates[0]) : new Date(),
```

Comment:
the same if is really bothering me ' can you
`[sd,ed]= dates.length? dates : new Array(2).fill(new Date()) or [new Date(),new Date()]`

Reply:
Done